quantumdb-core/src/main/java/io/quantumdb/core/schema/definitions/Table.java

```diff
@@ -133,14 +139,14 @@ public Index removeIndex(Collection<String> columns) {
 	}
 
 	public Index getIndex(String... columns) {
-		return getIndex(Sets.newHashSet(columns));
+		return getIndex(Lists.newArrayList(columns));
 	}
 
-	public Index getIndex(Collection<String> columns) {
+	public Index getIndex(List<String> columns) {
```

Comment:
Might be important to also specify if the index should be unique or not. But I am not sure what the added value would be when specifying two indexes on a table with the same order of columns with one unique and the other not.